docs/types.md

```diff
@@ -66,16 +66,13 @@ The non-alphanumeric characters cannot be repeated in a sequence (e.g. you can h
 Identifiers must start and end with alphanumeric characters.
 Identifiers are case insensitive (they are always stored in lower case).
 
-### Email
+### Course User Reference (CourseUserReference)
 
 Underlying Type: String
 
-An email address.
-
-#### Course Email Specification (CourseEmailSpec)
-
 When used in the context of a course,
-a `CourseEmailSpec` can be used to generalize email recipients.
+a `CourseUserReference` can be used to generalize targeting users
+(e.g, email recipients).
 The following values are allowed:
 
  - Email - Normal email addresses may be used.
```

Comment:
Put a sub-bullet here that mentions how the exact treatment of non-course emails may different depending on the use case (they may be ignored, used normally, result in an error, etc).